tests/unit/insertTestResult.unitTest.ts

```diff
@@ -83,6 +83,47 @@ describe("insertTestResult", () => {
                 });
         });
     });
+    context("when inserting a submitted, passing testResult without certificateNumber on TIR  type", () => {
+        it("should return 400-Bad request", () => {
+            const mockData = cloneDeep(testResultsMockDB[17]);
+            mockData.testTypes[0].testTypeId = "49";
+            MockTestResultsDAO = jest.fn().mockImplementation(() => {
+                return {
+                    insertTestResult: () => {
+                        return Promise.resolve({
+                            Items: Array.of(mockData),
+                            Count: 1
+                        });
+                    },
+                    getTestCodesAndClassificationFromTestTypes: () => {
+                        return Promise.resolve({
+                            linkedTestCode: "ttt",
+                            defaultTestCode: "tiv",
+                            testTypeClassification: "Annual With Certificate"
+                        });
+                    }
+                };
+            });
+
+            testResultsService = new TestResultsService(new MockTestResultsDAO());
+            for (const testType of mockData.testTypes) {
+                delete testType.testCode;
+                delete testType.testNumber;
+                delete testType.lastUpdatedAt;
+                delete testType.testAnniversaryDate;
+                delete testType.createdAt;
+                delete testType.testExpiryDate;
+                delete testType.certificateLink;
```

Comment:
Is it relevant to this test to remove all these fields?

Reply:
yeah, annoyingly.